OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -1683,7 +1683,7 @@ public void run() {
       runIdsAvailable.run();
    }
 
-   private static void fireIdsAvailableCallback() {
+   static void fireIdsAvailableCallback() {
```

Comment:
i commented in the github issue, but just curious. has this callback always been expected to fire twice?

Reply:
Yes, but only if there was an issue with the device getting a `registrationId`. More details in my comment on the #563 PR.